src/sensitivity_algorithms.jl

```diff
@@ -725,12 +725,12 @@ An implementation of the forward-mode, continuous
 allows for computing sensitivities of long-time averaged quantities with respect to the
 parameters of an `ODEProblem` by constraining the computation to the unstable subspace.
 `NILSS` employs the continuous-time `ForwardSensitivity` method as tangent solver. To
-avoid an exponential blow-up of the (homogenous and inhomogenous) tangent solutions,
+avoid an exponential blow-up of the (homogenous and inhomogeneous) tangent solutions,
```

Comment:
Is it really homogenous but inhomogeneous?

Reply:
It should be -geneous for both. But I'm not a native speaker.